src/hooks/useNotificationSound.ts

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from "react";
+import useSound from "use-sound";
+
+import notificationSound from "assets/sounds/newmessage.m4a";
+
+import { PrivateChatMessage, ChatTypes } from "types/chat";
+
+import { useChatSidebarControls } from "hooks/chatSidebar";
+
+import { useUser } from "./useUser";
+import { WithId } from "utils/id";
+
+export const useNotificationSound = (
```

Comment:
Given this is explicitly tied to private chats now, it might be better to rename it (both hook and file) to something like `usePrivateChatNotification`/`useNewPrivateChatNotification`/similar

Reply:
Part of me feels like it almost belongs in the `src/hooks/privateChats.ts` file too, instead of it's own file.